template/.github/workflows/build.yml.j2

```diff
@@ -9,7 +9,7 @@ on:
       - trying
       - "renovate/**"
     tags:
-      - "*"
+      - '[0-9][0-9].[0-9]+.[0-9]+'
```

Comment:
Is it on purpose that we require a two digit major number? Or am I misunderstanding the regex?
Also, does the . need to be escaped to match an actual . ?

Reply:
Well, we're at 22.11 already :) That's not a regex but a github-action specific glob pattern, for which there are only a limited number of special characters (otherwise we could have limited the second part to a number 1-12, for the month).